rules/windows/credential_access_dcsync_newterm_subjectuser.toml

```diff
@@ -0,0 +1,69 @@
+[metadata]
+creation_date = "2022/12/19"
+integration = ["windows"]
+maturity = "production"
+min_stack_comments = "New fields added: required_fields, related_integrations, setup, New Term rule type"
+min_stack_version = "8.4.0"
+updated_date = "2022/12/19"
+
+[rule]
+author = ["Elastic"]
+description = """
+This rule identifies when a User Account starts the Active Directory Replication Process for the first time. 
+Attackers can use the DCSync technique to get credential information of individual accounts or the entire domain, 
+thus compromising the entire domain.
+"""
+from = "now-9m"
+index = ["winlogbeat-*", "logs-system.*"]
+language = "kuery"
+license = "Elastic License v2"
+name = "FirstTime Seen Account Performing DCSync"
```

Comment:
```suggestion
name = "FirstTime Seen Account Performing DCSync"
note = """
## Setup

The 'Audit Directory Service Changes' logging policy must be configured for (Success, Failure).
Steps to implement the logging policy with Advanced Audit Configuration:

```
Computer Configuration >
Policies >
Windows Settings >
Security Settings >
Advanced Audit Policies Configuration >
Audit Policies >
DS Access >
Audit Directory Service Changes (Success,Failure)
```

If enabling an EQL rule on a non-elastic-agent index (such as beats) for versions <8.2, events will not define `event.ingested` and default fallback for EQL rules was not added until 8.2, so you will need to add a custom pipeline to populate `event.ingested` to @timestamp for this rule to work.
"""
"""
```

Reply:
There is a conflict on this one in the ``` usage